openeo/internal/process_graph_visitor.py

```diff
@@ -116,6 +116,10 @@ def _accept_process(self, process_id: str, arguments: dict, namespace: Union[str
 
     def _accept_argument_list(self, elements: list):
         for element in elements:
+            if isinstance(element,dict) and "from_parameter" in element:
+                if element["from_parameter"] in self.parameters:
+                    element = self.parameters[element["from_parameter"]]
```

Comment:
I guess something is missing: `self.parameters` does not exist yet here in this PR

Reply:
see above, I think it does exist (if I'm not overlooking something)